### ISSUE TITLE
`unicorn/isolated-functions` - Support methods

### ISSUE THREAD

**Issue by fregante** [sindresorhus/eslint-plugin-unicorn#3463]
Reported ✅

```ts
const lol = 1;

/** @isolated */
export function fn() {
	console.log(lol)
}
```

Not reported ❌

```js
const lol = 1;

export const object = {
	/** @isolated */
	method() {
		console.log(lol)
	}
}
```
